modules/katamari/src/main/ts/ephox/katamari/api/Maybe.ts

```diff
@@ -0,0 +1,6 @@
+import { Just, Nothing } from './Maybes';
+
+// The Maybe type, representing a value that either does or doesn't exist. For
```

Comment:
What's our standard on function comments here? I've seen a javadoc-style in other places.

Reply:
Yeah, this really should be using `/** */` style comments instead.